scalameta/inputs/shared/src/main/scala/scala/meta/inputs/Position.scala

```diff
@@ -41,4 +41,17 @@ object Position {
     override def text = new String(input.chars, start, end - start)
     override def toString = s"[$start..$end) in $input"
   }
+
+  object Range {
+    def apply(
+        input: Input,
+        startLine: Int,
+        startColumn: Int,
+        endLine: Int,
+        endColumn: Int): Position.Range = {
+      val start = input.lineToOffset(startLine) + startColumn
```

Comment:
👍 I already implemented this in Scalafix and I agree it should be in the public API. 

However, we should decide what to do when the column numbers exceed the column width in the actual input. This implementation is not LSP compliant, where `column=Int.MaxValue` will translate to the largest possible column for the corresponding line number. I opened https://github.com/scalameta/scalameta/issues/1728 to discuss this better.

Reply:
Great catch!! Thank you for following up!